manual/root_macros_and_shared_libraries/index.md

```diff
@@ -136,14 +122,13 @@ library by using `#include` statements; you will need to compile each ROOT macro
 ### Compiling a ROOT macro with ACLiC
 
 Before you can compile your interpreted ROOT macro, you need to add the include statements for
-the classes used in the ROOT macro. Only then you can build and load a shared library containing
-your ROOT macro.
+the classes used in the ROOT macro. Only then you can build and load a shared library containing your ROOT macro.
 
 You can compile a ROOT macro with:
 
   - default optimizations
 
-  - optimizations
+  - different optimizations
 
   - debug symbols
 
```

Comment:
One comment about the note. 

You can compile a ROOT macro that has the function main() but when you execute you cannot call the main() function. 
We should maybe re-phrase this like: 

When a ROOT macro has a function called `main()` your can compile the macro with ACLiC or the Cling ROOT interpreter, but you cannot execute the `main()` function from within the ROOT session.

Reply:
As a side note I find this a useful functionality Because you can execute at the same time a macro from ROOT and outside ROOT compiling with `root-config`  by doing:
```
void MyMacro() {
.........
} 
int main()  {
  MyMacro();
}
```

Reply:
I would also move this note at the end of the sub-paragraph after describing the different way of compiling with ACLic a shared library

Reply:
Good to know!